src/index.js

```diff
@@ -105,7 +105,9 @@ class Radar {
 
         // Setup http
         const headers = {
-          Authorization: publishableKey
+          Authorization: publishableKey,
+          'X-Radar-SDK-Version': SDK_VERSION,
+          'X-Radar-Device-Type': 'Web',
```

Comment:
Let's set the default headers in `http.js`. We can merge them with whatever is passed in.